tests/_helpers/github-workflows.js

```diff
@@ -0,0 +1,70 @@
+const GitHubClient = require( './github-client' );
+
+function verifyWorkflowStatus( workflowObject, reportCurrentStatus, waitingTime ) {
+	return new Promise( ( resolve, reject ) => {
+		if( workflowObject.status === 'completed' ) {
+			resolve( workflowObject );
+			return;
+		}
+		waitingTime = waitingTime || 1000;
+
+		reportCurrentStatus( workflowObject, waitingTime );
+
+		setTimeout( async () => {
+			const workflow = await getWorkflowRun( workflowObject.id );
+			// Give some time between rechecks
+			const result = await verifyWorkflowStatus( workflow.data, reportCurrentStatus, waitingTime + 3500 );
+			resolve( result );
+		}, waitingTime );
+	} );
+}
+
+async function getRunningActions() {
```

Comment:
I guess it will be good to stick to one name so `Workflow` vs `Action`. From what I see `Workflow` should be used :thinking: